website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,237 @@
+
+
+# Roles
+ A Role is a collection of permissions that defines access for entities or resources which can be further assigned to a user or a group. Roles allow you to adjust multiple permissions rather than assigning individual permissions to users/groups. In Appsmith, a role can contain multiple permissions and a permission can be present in multiple roles. However, a role can't be nested, i.e, it can contain only permissions, not other roles. 
+ 
+ In an Appsmith instance, there are three roles available by default -
```

Comment:
Let's not include numbers, we can just say below or following instead. Also, do we need to add `Instance` here?

Reply:
Yes, we need to mention it here as all of this will happen on an instance

Reply:
> I think we should mention the three roles thing because then we want to inform that GAC allows users to assign a custom role beyond these three.
> 
> Also give a screenshot here so users know exactly where they will be able to assign this. <img alt="Screenshot 2022-12-23 at 11 21 32" width="374" src="https://user-images.githubusercontent.com/32227135/209279437-4ce4bc4b-aa33-4cdf-b09f-df83680311d2.png">

We should not use numbers to refer to default roles because we may add more roles to the default list in the future. The phrase 'by default' means that these roles are currently available, and three of them exist.